sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/ln/LN0Adapter.java

```diff
@@ -237,13 +234,23 @@ public List<SclReportItem> updateDoInRef() {
                         && doiAdapter.findDataAdapterByName(DAI_NAME_PURPOSE).isPresent())
                 .map(doiAdapter -> doiAdapter.getDataAdapterByName(DAI_NAME_PURPOSE).getCurrentElem().getVal().stream()
                         .findFirst()
-                        .map(tVal -> doiAdapter.updateDaiFromExtRef(getBoundExtRefsByDesc(tVal.getValue())))
+                        .map(tVal -> doiAdapter.updateDaiFromExtRef(getExtRefsBoundToInRef(tVal.getValue())))
                         .orElse(List.of(SclReportItem.warning(getXPath(), "The DOI %s can't be bound with an ExtRef".formatted(getXPath()))))
                 )
                 .flatMap(List::stream)
                 .collect(Collectors.toList());
     }
 
+    private List<TExtRef> getExtRefsBoundToInRef(String desc) {
+        List<TExtRef> boundExtRefs = getBoundExtRefsByDesc(desc);
+        // Special case for DIGITAL CHANNEL of type BOOLEAN in LDEVICE LDEPF RSR-1048
+        if (boundExtRefs.isEmpty() && getParentLDevice().getInst().equals("LDEPF") && desc.contains("DIGITAL") && desc.endsWith("_BOOLEAN")) {
```

Comment:
Exact criteria to be confirmed with @hkaroun

Reply:
BOOLEAN